### PR TITLE
VB-3: Feature/connect open api tool

### PR DESCRIPTION
**Pull request by mishabenq**
image

### CODE REVIEW

**Review by Nightlngale — CHANGES_REQUESTED** [review]
Routes !== Repository methods

File: packages/backend/src/repositories/types/CreateRefreshToken.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type CreateRefreshToken = {
```
Comment (d-uvin) [inline packages/backend/src/repositories/types/CreateRefreshToken.ts:1]: we kebab-case in files namings

Comment (dmitriy-uvin) [inline packages/backend/src/repositories/types/CreateRefreshToken.ts:1]: we use kebab-case in files namings

File: packages/backend/src/repositories/types/CreateUser.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+type CreateUser = {
```
Comment (dmitriy-uvin) [inline packages/backend/src/repositories/types/CreateUser.ts:1]: also file name must use kebab-case

File: packages/backend/src/server.ts [hunk 3]
```diff
@@ -5,6 +5,10 @@ import { initRepositories } from '@repositories';
 import { getEnv, logger } from '@helpers';
 import { initServices } from '@services';
 import { loggerMiddleware, localizationMiddleware } from '@middlewares';
+import swaggerUi from 'swagger-ui-express';
+// import swaggerJsdoc from 'swagger-jsdoc';
```
Comment (dmitriy-uvin) [inline packages/backend/src/server.ts:9]: remove comments

File: packages/backend/src/server.ts [hunk 4]
```diff
@@ -14,14 +18,29 @@ const services = initServices(repositories);
...
+//   definition: {
+//     openapi: '3.0.0',
+//     info: {
+//       title: 'Hello World',
+//       version: '1.0.0',
+//     },
+//   },
+//   apis: ['**/*.ts'], // files containing annotations as above
+// };
+// const swaggerSpecification = swaggerJsdoc(options);
```
Comment (dmitriy-uvin) [inline packages/backend/src/server.ts:31]: comments

**Comment by mishabenq** [comment]
image

File: packages/backend/tsconfig.json [hunk 5]
```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "baseUrl": "src",
     "module": "commonjs",
+    "experimentalDecorators": true,
```
Comment (Vitalii4as) [inline packages/backend/tsconfig.json:6]: We don't need it anymore

File: packages/backend/package.json [hunk 6]
```diff
@@ -38,6 +38,9 @@
     "node-fetch": "^2.6.7",
     "redis": "^4.2.0",
     "sib-api-v3-typescript": "^2.2.2",
+    "swagger-jsdoc": "^6.2.5",
+    "swagger-ui-express": "^4.5.0",
+    "tsoa": "^4.1.1",
```
Comment (Vitalii4as) [inline packages/backend/package.json:43]: Remove tsoa, dist and and tsoa.json

**Review by Vitalii4as — CHANGES_REQUESTED** [review]

**Review by Vitalii4as — APPROVED** [review]

**Pull request merged**